### ISSUE TITLE
snippet on words includes punctuation

### ISSUE THREAD

**Issue by NikhilTilak** [data-dart/bookend#20]
snippet should also use the tokenize method with punctuation removal. Once @emapple 's new pull request is merged we can use the same code in snippet.

![Screen Shot 2020-05-16 at 11 03 38 AM](https://user-images.githubusercontent.com/49879990/82122994-f3bd1380-9764-11ea-8bb3-31b4bd2d6e8c.png)

**Comment by emapple** [comment]
You mean for char and word-based snippets, right? This wouldn't make sense for sentence-based snippets.

We can add it as an option in snippet (i.e., `include_punctation=False`)

Edit: I'm an idiot and didn't fully read the issue title
